includes/wp-admin/class-health-check.php

```diff
@@ -432,6 +432,18 @@ public static function test_pretty_permalinks() {
 					esc_url( admin_url( 'options-permalink.php' ) )
 				)
 			);
+		} elseif ( str_starts_with( $permalink_structure, '/index.php' ) ) {
+			$result['status']         = 'critical';
+			$result['label']          = \__( 'ActivityPub does not support the "Almost Pretty Permalinks" structure.', 'activitypub' );
```

Comment:
AI suggestion: "Your permalink structure needs to be updated for ActivityPub to work properly."